tw-tkms-starter/src/main/java/com/transferwise/kafka/tkms/config/TkmsKafkaProducerProvider.java

```diff
@@ -39,8 +42,12 @@ public class TkmsKafkaProducerProvider implements ITkmsKafkaProducerProvider, Gr
 
   private Map<Pair<TkmsShardPartition, UseCase>, ProducerEntry> producers = new ConcurrentHashMap<>();
 
+  @Autowired
+  private List<ITkmsKafkaProducerPostProcessor> postProcessors = new ArrayList<>();
```

Comment:
Do you think there's likely to be a situation where there will be multiple `ITkmsKafkaProducerPostProcessor`s in the Spring context and we're interested in defining their ordering? If the extensions are being provided by other libraries I'm not sure how we'd do that in the auto configuration.